Update Readme with `inwer` information (#51)
sio2project/sinol-make: README.md

of CPUs. Measures the solutions' time with oiejq, unless specified otherwise. After running the solutions, it
compares the solutions' scores with the ones saved in config.yml.
Run `sinol-make run --help` to see available flags.

> - `sinol-make inwer` -- Verifies if input files are correct using inwer program in parallel. You can specify what inwer
> program to use, what tests to check and how many CPUs to use. Run `sinol-make inwer --help` to see available flags.

Reviewer: ```suggestion
- `sinol-make inwer` -- Verifies whether input files are correct using your "inwer.cpp" program. You can specify what inwer
program to use, what tests to check and how many CPUs to use. Run `sinol-make inwer --help` to see available flags.
```
